s3_bucket.go

```diff
@@ -1323,3 +1325,64 @@ func (s *s3ArchiveBucket) Pull(ctx context.Context, opts SyncOptions) error {
 
 	return nil
 }
+
+// PresignExpireTime sets the amount of time the link is live before expiring.
+const PresignExpireTime = 24 * time.Hour
+
+// RequestParams holds all the parameters needed to sign a url or fetch headObject.
+type RequestParams struct {
+	Bucket    string `json:"bucket"`
+	FileKey   string `json:"fileKey"`
```

Comment:
Nit: Can we use snake case for the json tags? And for consistency change `FileKey` to `Key`.

Reply:
Since this is already being used in prod, I don't think it's a good idea to change it now